### PR TITLE
make table with responsibilities

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
I made some updates to the Mondo project management doc, I created a table that lists who is responsible. 

Let me know your thoughts!

You can see the previous draft here: https://mondo.readthedocs.io/en/latest/editors-guide/project-management/

### CODE REVIEW

File: docs/editors-guide/project-management.md [hunk 1]
```diff
@@ -19,59 +19,23 @@ Role | Description | Person(s) | Relevant issues
...
+Advertising workshops | Workshops should be advertised on our [website](https://mondo.monarchinitiative.org/pages/workshop/), on our Slack channel, on the mailing list | Project Manager
+Set milestones and triage tickets |All [tickets](https://github.com/monarch-initiative/mondo/issues) should be: 1. Acknowledged, thank the contributor for the ticket, 2. Assigned to a curation or technical team member, 3. Assigned to a monthly milestone, 4. Given a label | Lead Curator
+Mondo Releases |Mondo should be released on a monthly basis on/around the first of the month. Please refer to the [pre-release checklist](https://mondo.readthedocs.io/en/latest/editors-guide/pre-release-checklist/) for necessary tasks before the release. Instructions are [here](https://mondo.readthedocs.io/en/latest/developer-guide/release/). If there are any delays with the release (more than a couple days), please communicate these to the Mondo users on the mailing list and in Slack. Some of our users depend on the regular releases, such as EFO. | Curation team
```
Comment (matentzn) [inline docs/editors-guide/project-management.md:34]: I would never assign responsibilities to teams. The responsibility should lie in a person - who makes the release is secondary (i.e. as curation lead, you can assign someone).

File: docs/editors-guide/project-management.md [hunk 2]
```diff
@@ -19,59 +19,23 @@ Role | Description | Person(s) | Relevant issues
...
+Grant reporting | 1. Link Mondo tickets to [Phenomics First Operations Board](https://github.com/monarch-initiative/phenomics_first_resource/issues?q=is%3Aissue+is%3Aopen+label%3A%22Aim+2+-+MONDO%22) 2. Ensure deliverables for P1 grant are being addressed, 3. Send all dissemination artifacts to the 'nuggets' address for Monarch/TISLab (to help RPPR prepping) | Project Manager
+RPPR | Coordinate reporting for yearly RPPR | Project Manager
+Website updates |The [Mondo website](https://mondo.monarchinitiative.org/) should be regularly updated (instructions are [here](https://mondo.readthedocs.io/en/latest/editors-guide/mondo-website-editing/)). Regular updates that may be needed include: [Mondo users](https://mondo.monarchinitiative.org/pages/users/), [Mondo contributors](https://mondo.monarchinitiative.org/pages/contributors/), [Resources](https://mondo.monarchinitiative.org/pages/resources/)- including new papers, presentations, blog posts, etc., [Workhops](https://mondo.monarchinitiative.org/pages/workshop/), [FAQs](https://mondo.monarchinitiative.org/pages/faq/) should be updated as new questions arise | Curation team + Project Manager
```
Comment (matentzn) [inline docs/editors-guide/project-management.md:37]: Same here, I would only add who is _responsible_ for making the edits, not who makes the edits.

File: docs/editors-guide/project-management.md [hunk 3]
```diff
@@ -19,59 +19,23 @@ Role | Description | Person(s) | Relevant issues
...
+Liaise with Mondo Users | Keep track of Mondo users (keep contact info up to date, what project they are working on, how they are using Mondo) | Curation team
```
Comment (matentzn) [inline docs/editors-guide/project-management.md:38]: see above.

**Review by matentzn — COMMENTED** [review]
Looks great! One small comment (3 times) that's all!

File: docs/editors-guide/project-management.md (hunk 1 above)
Comment (nicolevasilevsky) [inline docs/editors-guide/project-management.md:34]:
```suggestion
Mondo Releases |Mondo should be released on a monthly basis on/around the first of the month. Please refer to the [pre-release checklist](https://mondo.readthedocs.io/en/latest/editors-guide/pre-release-checklist/) for necessary tasks before the release. Instructions are [here](https://mondo.readthedocs.io/en/latest/developer-guide/release/). If there are any delays with the release (more than a couple days), please communicate these to the Mondo users on the mailing list and in Slack. Some of our users depend on the regular releases, such as EFO. | Lead curator
```

File: docs/editors-guide/project-management.md [hunk 4]
```diff
@@ -19,59 +19,23 @@ Role | Description | Person(s) | Relevant issues
...
+RPPR | Coordinate reporting for yearly RPPR | Project Manager
+Website updates |The [Mondo website](https://mondo.monarchinitiative.org/) should be regularly updated (instructions are [here](https://mondo.readthedocs.io/en/latest/editors-guide/mondo-website-editing/)). Regular updates that may be needed include: [Mondo users](https://mondo.monarchinitiative.org/pages/users/), [Mondo contributors](https://mondo.monarchinitiative.org/pages/contributors/), [Resources](https://mondo.monarchinitiative.org/pages/resources/)- including new papers, presentations, blog posts, etc., [Workhops](https://mondo.monarchinitiative.org/pages/workshop/), [FAQs](https://mondo.monarchinitiative.org/pages/faq/) should be updated as new questions arise | Lead curator
+Liaise with Mondo Users | Keep track of Mondo users (keep contact info up to date, what project they are working on, how they are using Mondo) | Curation team
```
Comment (nicolevasilevsky) [inline docs/editors-guide/project-management.md:38]:
```suggestion
Liaise with Mondo Users | Keep track of Mondo users (keep contact info up to date, what project they are working on, how they are using Mondo) | Lead curator
```

**Pull request merged**